src/Client/Web/Bit.Client.Web.BlazorUI/Components/Rating/BitRating.razor.cs

```diff
@@ -49,6 +65,7 @@ protected override async Task OnInitializedAsync()
             min = AllowZeroStars == true ? 0 : 1;
             Max = Max > min ? Max : min;
             DefaultRating = DefaultRating > 0 ? DefaultRating : min;
+            ratingValue = DefaultRating > 0 ? DefaultRating : ratingValue;
```

Comment:
are you sure about this line?
1. I think in this line the ratingValue should be the base of comparison: ratingValue = ratingValue > 0 ? ...
2. This line is changing the internal value of a 2-way bound parameter, which is not right

Reply:
I think , if rating component has default value, for example 3, its rating value should be 3 too on Initialized method.